scenes/game/game.gd

```diff
@@ -821,6 +832,9 @@ func _update_clocks():
 	if observer_mode: return
 	$PlayerLife.set_clock(player_clock_remaining)
 	$OpponentLife.set_clock(opponent_clock_remaining)
+	if enforce_timer and player_clock_remaining <= 0:
+		game_wrapper.trigger_timeout(Enums.PlayerId.PlayerId_Player, 
```

Comment:
rather than trigger_timeout, maybe trigger_clockranout to go with renaming the enum.